packages/cmf-router/package.json

```diff
@@ -3,11 +3,26 @@
   "version": "6.1.2",
   "description": "",
   "main": "lib/index.js",
+  "module": "./lib-esm/index.js",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": {
+        "types": "./lib-esm/index.d.ts",
+        "default": "./lib-esm/index.js"
```

Comment:
if there is a `.d.ts` with the same name next to a `.js` name it will be picked up automatically by typescript and code editors. So this can be simplified to:

```
".": {
      "import": "./lib-esm/index.js",
      "require": "./lib/index.js"
    }
```